Lib/fontTools/feaLib/parser.py

```diff
@@ -96,16 +111,18 @@ def parse(self):
         return self.doc_
 
     def parse_anchor_(self):
```

Comment:
I think the meaning of the `_` at the end was to signal that the method is private (although usually the convention is to put it at the beginning). We don't need to document private methods. How would a client go on and use these in a meaningful way? They simply call `parse` and let the Parser do its job, rather than call specific parse_something_ methods.
Maybe @brawer can confirm this.

Reply:
@simoncozens maybe you can turn those into inline comments `#` instead of docstrings `"""` as they may still be useful for developers looking at the code (not to the users reading at docs)

Reply:
Many of the private methods had docstrings before I started!
https://github.com/fonttools/fonttools/blob/089f24da6b30cce4e24775a84a4497815ae72d28/Lib/fontTools/feaLib/parser.py#L1670-L1684

If we set the public API either through the `__all__` variable or in Sphinx, so that only those methods appear in the docs, I don't think it's a problem that private methods have docstrings. In fact, it helps with doc coverage testing to have a docstring rather than a comment.